hubblestack/files/hubblestack_nova/misc.py

```diff
@@ -91,7 +91,7 @@ def audit(data_list, tags, debug=False, **kwargs):
                         ret['Errors'] = []
                     ret['Errors'].append({tag: 'No function {0} found'
                                               .format(tag_data['function'])})
-                    continue
+		    continue
```

Comment:
Can you please make sure indentation is correct ? As in PR it looks like shifted a bit.